Os/Posix/Task.cpp

```diff
@@ -100,7 +100,9 @@ namespace Os {
 
     Task::TaskStatus set_cpu_affinity(pthread_attr_t& att, NATIVE_UINT_TYPE cpuAffinity) {
         if (cpuAffinity != Task::TASK_DEFAULT) {
-#ifdef TGT_OS_TYPE_LINUX
+#ifdef TGT_OS_TYPE_LINUX 
```

Comment:
You should move this include to the top.  Including a header inside code should be avoided.

Reply:
That's a much cleaner way of doing it, thanks